src/include/miopen/env_debug.hpp

```diff
@@ -46,6 +46,29 @@ MIOPEN_INTERNALS_EXPORT void UpdateEnvVariable(std::string_view name, std::strin
 // MT-Unsafe
 MIOPEN_INTERNALS_EXPORT void ClearEnvVariable(std::string_view name);
 
+class EnvVarSanitizer
```

Comment:
similar class is here https://github.com/ROCm/MIOpen/blob/develop/test/gtest/gtest_common.hpp#L43. Can we reuse that class?